rocketpy/Rocket.py

```diff
@@ -449,6 +442,59 @@ def addMotor(self, motor, position):
         self.evaluateStaticMargin()
         return None
 
+    def addSurface(self, surface, position):
+        """Adds an aerodynamic surface to the rocket. The aerodynamic surface
+        must be an instance of a class that inherits from the AerodynamicSurface
+
+        Parameters
+        ----------
+        surface : AerodynamicSurface, NoseCone, TrapezoidalFins, EllipticalFins, Tail
+            Aerodynamic surface to be added to the rocket. See AerodynamicSurface class
+            for more information.
+        position : int, float
+            Position, in m, of the aerodynamic surface's center of pressure relative to
+            the user defined rocket coordinate system. See `Rocket.coordinateSystemOrientation`
+            for more information regarding the rocket's coordinate system.
+
+        Returns
+        -------
+        None
+        """
+        self.aerodynamicSurfaces.append(surface)
+        surface.position = position
+        if isinstance(surface, NoseCone):
+            self.nosecone.append(surface)
+        elif isinstance(surface, (TrapezoidalFins, EllipticalFins)):
+            self.fins.append(surface)
+        elif isinstance(surface, Tail):
+            self.tail.append(surface)
+        self.evaluateStaticMargin()
+        return None
+
+    def addSurfaces(self, surfaces, positions):
```

Comment:
Raising the discussion referenced in the PR description, personally I believe that these two methods have too similar names and functionalities. It does not seem _pythonic_ to slightly differentiate methods naming if the parameter is a single item or a list, the method `addSurface` should take care of both cases, maybe with an `try-except` clause or `if-else`.